main.py

```diff
@@ -65,9 +66,16 @@ def change_read():
     book.read = new_read 
     book.save()
     
+def delete_book():
+    try:
+        search_id = ui.ask_question('Enter Book ID:')
+        match_id = store.get_book_by_id(search_id)
+        store._delete_book(match_id)
```

Comment:
Don't call any _ methods directly. There's another method that's already in bookstore to use.